### PR TITLE
Small tweaks to the last PR #404

### PR DESCRIPTION
**Pull request by Cybolic**
Added extra padding to larger sizes, fixed initial padding and reordered code to always go from small to large.

### CODE REVIEW

**Comment by Cybolic** [comment]
I just renamed the stock class, but I can name it back if you prefer.

**Comment by JasonLG1979** [comment]
What I mean is that popup-menu-icon already has padding in the class. You don't need to add a custom class for small.

**Comment by Cybolic** [comment]
Done :)

**Comment by Cybolic** [comment]
To clarify, I kept the padding in the small/default class to make sure we have consistent paddings across the three sizes, regardless of the theme.

**Comment by JasonLG1979** [comment]
To consistency means using as little custom classes as possible as not to clash with the present theme.

**Comment by JasonLG1979** [comment]
*to me

**Comment by Cybolic** [comment]
Seeing as we're using the class for popup menu icons to style playback buttons (ie. using a class for something it wasn't intended for), I think we need to restrict the theme a bit to make sure it works for our use.

**Comment by JasonLG1979** [comment]
> Seeing as we're using the class for popup menu icons to style playback buttons (ie. using a class for something it wasn't intended for), I think we need to restrict the theme a bit to make sure it works for our use.
I'm all about custom classes for med and large because we really don't have a choice but I disagree with overriding the padding of popup-menu-icon.

**Comment by Cybolic** [comment]
Well, the padding override for `popup-menu-icon` was already in the code before the PR; as it is now, the PR just adds the classes for medium and large.

**Comment by JasonLG1979** [comment]
> Well, the padding override for popup-menu-icon was already in the code before the PR; as it is now, the PR just adds the classes for medium and large.

No it wasn't. Look again.

**Comment by Cybolic** [comment]
Ah, right you are! I was looking at the class itself and forget the override itself. I've removed it now.

File: src/stylesheet.css [hunk 1]
```diff
@@ -1,6 +1,12 @@
 .player-button {
     padding: 0 4px;
 }
```
Comment (JasonLG1979) [inline src/stylesheet.css]: Don't need the player-button class. It's not used by anything git rid of it and I'll merge this PR.

**Pull request merged**